### DISCUSSION TITLE
No-code front-end builder

### DISCUSSION THREAD

**Discussion by FlavioSuarez** [abydahana/aksara#475] [Ideas]
Hi,

It would be great if Aksara would have a no-code (visual) builder for the front-end, as [microweber](https://microweber.org) has...
Microweber is made on Laravel. I'd love to see the same features made on CodeIgniter!

Just my 0.02
Best wishes!

**Comment by abydahana** [comment]
If only there's such similar editor

**Comment by FlavioSuarez** [comment]
The free website builder [Mobirise](https://mobirise.com) has the same approach.
I love it as a tool for non developers.
Yes, I can see it's a lot of work to build something like this!
